howtos/cmake_launch.rst

```diff
@@ -42,4 +42,4 @@ If you want to use targets, you could do:
                     BASIC_SETUP CMAKE_TARGETS)
```

Comment:
Use `catch2` from Conan Center? (for a different issue/PR)

Reply:
we have to review all the examples from the documentation so...